src/index.ts

```diff
@@ -94,6 +95,18 @@ export const resolveFetch = () => {
     return undefined;
 };
 
+const resolveAbortController = () => {
+    try {
+        if (typeof window !== 'undefined' && 'AbortController' in window) {
+            return () => new window.AbortController();
+        } else if ('fetch' in globalThis) {
+            return () => new globalThis.AbortController();
+        }
+    } catch (e) {
+        console.error('Unleash failed to resolve "AbortController" factory', e);
```

Comment:
This should basically never happen, right? This is if someone is doing something unusual like swapping out fetch?

Reply:
Correct - someone would need to take deliberate effort to end up in this situation